javatests/arcs/core/data/expression/ExpressionTest.kt

```diff
@@ -239,6 +239,17 @@ class ExpressionTest {
         ).isEqualTo(numbers.map { 5.5 })
     }
 
+    @Test
+    fun evaluate_paxel_average_onComplexExpression() {
+        val selectAvgExpr = average(
+            from<Number>("p") on "numbers"
+            select currentScope["p"].asNumber() + 10.asExpr()
+        )
+        assertThat(
+            evalExpression<Number>(selectAvgExpr, currentScope)
+        ).isEqualTo(15.5)
```

Comment:
maybe change to `numbers.map { it + 10 }.average()` ?

Reply:
Done